variables.tf

```
@@ -18,8 +18,12 @@ variable "include_tags" {
 
 variable "monitors" {
   type = map(object({
-    name       = string
-    message    = string
+    name    = string
+    message = string
+    monitor_threshold_windows = object({
```

As this field is already in the `monitors` object, I think we can skip the `monitor` part here and just call this:
```suggestion
   threshold_windows = object({
```
It should also be reordered after the rename...

---

I agree, considering `thresholds` is also there we should keep it consistent.